src/components/profile/profile-edit-modal.tsx

```diff
@@ -110,7 +112,7 @@ export const ProfileEditModal: React.FC<ProfileEditModalProps> = ({
         firstName: data.firstName,
         lastName: data.lastName,
         city: data.city,
-        tags: skills,
+        tags: data.tags,
         bio: data.bio,
```

Comment:
```suggestion
        firstName: data.firstName,
        ...data
```

Since all of the names are the same, you can just destructure this. Also, a more descriptive name like `formData` would be good.

Reply:
will do!